api/controllers/location.js

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+const mongoose = require('mongoose')
+require('../models/Sample') // Register model
+const Sample = mongoose.model('Sample')
+
+const DEFAULT_RSSI = 0
+const K1 = 10
+const K2 = 10
+
+module.exports = {
+  getLocation: async (req, res) => {
+    const buildingId = req.body._id
+    const query = Sample.find({ buildingId: buildingId })
+    query.lean()
+
+    try {
+      const samples = await query.exec()
+
+      if (samples === null) {
+        return res.status(404).json({})
```

Comment:
```suggestion
        return res.status(404)
```

El JSON vacío no aporta nada, o pongamos un mensaje de error o no mandemos nada